src/index.html

```diff
@@ -0,0 +1,194 @@
+              Description
+              <input
+                id="profile-description-input"
+                type="text"
+                class="modal__input"
+                placeholder="Describe yourself"
+                required
+                minlength="2"
+                maxlength="200"
+              />
+              <span class="modal__error" id="profile-description-input-error"
+                >Please lengthen this text to 2 characters or more (you are
+                currently using 1 character).</span
+              >
+            </label>
+            <button type="submit" class="modal__submit-btn">Save</button>
+          </form>
+        </div>
+      </div>
+    </div>
+    <div class="modal" id="add-card-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-btn"></button>
+        <h2 class="modal__title">New Post</h2>
+        <form name="edit-profile" class="modal__form" novalidate>
+          <label for="add-card-link-input" class="modal__label">
+            Image Link
+            <input
+              id="add-card-link-input"
+              type="url"
+              name="name"
+              class="modal__input"
+              placeholder="Paste a link to the picture"
+              required
+            />
+            <span id="add-card-link-input-error" class="modal__error"></span>
+          </label>
+          <label for="add-card-name-input" class="modal__label">
+            Caption
+            <input
+              id="add-card-name-input"
+              type="text"
+              name="name"
+              class="modal__input"
+              placeholder="Type your caption"
+              required
+              minlength="2"
+              maxlength="30"
+            />
+            <span id="add-card-name-input-error" class="modal__error"></span>
+          </label>
+          <button type="submit" class="modal__submit-btn">Save</button>
+        </form>
+      </div>
+    </div>
+    <div class="modal" id="preview-modal">
+      <div class="modal__container modal__container_type_preview">
+        <button
+          type="button"
+          class="modal__close-btn modal__close-btn_type_preview"
+        ></button>
+        <img class="modal__image" src="" alt="" />
+        <p class="modal__caption"></p>
+      </div>
+    </div>
+
+    <div class="modal" id="edit-avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-btn"></button>
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form name="edit-profile" class="modal__form" novalidate>
+          <label for="profile-avatar-input" class="modal__label">
+            Profile Picture
+            <input
+              id="profile-avatar-input"
+              type="text"
+              class="modal__input"
+              placeholder="Paste the link to an image"
+              required
+              minlength="2"
+              maxlength="40"
```

Comment:
* I can't change the avatar. Please, remove `maxlength` because the links can be very long